### DISCUSSION TITLE
Force re-upload of broken assets on Immich?

### DISCUSSION THREAD

**Discussion by bbklopfer** [immich-app/immich#3714] [Q&A] [answered]
I haven't been able to find the proper way to force a re-upload of an image from mobile --- any suggestions?

The images in question are from an iOS device. They show up correctly on the iOS device (both native Photos app and Immich), and claim to be uploaded (cloud w/check mark icon). On Android and web, they do not show up. If I try to download the image on web, it fails, with an immich_server log message of

`ERROR [ExceptionsHandler] ENOENT: no such file or directory, stat 'upload/library/admin/path/to/file.jpg'`

I've read it's possible to fix these issues with some Postgres magic, but I've also read that that is Strongly Discouraged.

I believe the original issue of why the files got borked was I didn't have a sufficient `client_max_body_size` set (I'm using a reverse proxy, nginx). This is just a hunch though...

Thanks in advance!

**Comment by bbklopfer** [comment, marked as answer]
Ok, going off of some comments in https://github.com/immich-app/immich/discussions/3192 I tried deleting from the web interface, and this seems to work --- iOS client seems to re-upload, fixing the issue.

Would be nice to be able to batch verify, but for now this works.

BTW --- absolutely amazing software, very, very impressed!

**Reply by alextran1502** [reply]
Thank you, we still need to harden the upload process, there is some race condition that has yet to be resolved and that leave those orphaned files

**Reply by kingp0dd** [reply]
Ran into the same issue. Is there a batch verify function that checks all missing files and reupload them from phone?

**Comment by Barrow1990** [comment]
Has there been a fix for this to get a bulk remove or something. I have loads from multiple users which im unable to delete as its stating: `Not found or no asset.delete access (Immich Server Error)`

**Comment by electricmessiah** [comment]
Just to clarify.. the process is:

1. verify the photos are on your phone or device
2. delete the error photos from the timeline in Immich
3. empty or remove the specific photos from the trash in Immich
4. restart the app and it should find the photos and reupload them

This only happens to me when I forget to upgrade the remote Immich instance I run for CUDA processing. If I forget to update the remote instance after updating the main instance, it will sometimes create errored uploads (when it should be falling back to local CPU processing) and this process must be invoked to fix it.
